pcdsdevices/epics_motor.py

```diff
@@ -51,6 +51,8 @@ class PCDSMotorBase(FltMvInterface, EpicsMotor):
     # Disable missing field that our EPICS motor record lacks
     # This attribute is tracked by the _pos_changed callback
     direction_of_travel = Component(Signal)
+    # This attribute will show the current status of motor
```

Comment:
This comment doesn't make any sense. The "disable puts" field has little to do with the motor's status.